libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -399,22 +400,72 @@ public boolean isActionInProgress() {
         return System.currentTimeMillis() - mActionStartedAt < MAX_ACTION_TIME_MS;
     }
 
-    private void updateFailedMediaList() {
-        AztecText.AttributePredicate failedPredicate = new AztecText.AttributePredicate() {
+    private void updateUploadingMediaList() {
+        AztecText.AttributePredicate uploadingPredicate = getPredicateWithClass("uploading");
+
+        mUploadingMediaProgressMax.clear();
+
+        // update all items with upload progress of zero
+        for (Attributes attrs : content.getAllElementAttributes(uploadingPredicate)) {
+            String localMediaId = attrs.getValue("data-wpid");
+            if (!TextUtils.isEmpty(localMediaId)) {
+                mUploadingMediaProgressMax.put(localMediaId, new Float(0));
```

Comment:
We can drop an unnecessary boxing here and make a tiny optimization by just using `0F` instead of `new Float(0)` (also clears a lint warning).